csrankings-r.csv

```diff
@@ -443,6 +444,7 @@ Rebecca J. Passonneau,Pennsylvania State University,https://www.eecs.psu.edu/dep
 Rebecca M. Willett,University of Chicago,https://voices.uchicago.edu/willett,bGRVPl8AAAAJ
 Rebecca N. Wright,Columbia University,https://www.cs.columbia.edu/~rwright,L-xeJPoAAAAJ
 Rebecca Willett,University of Chicago,https://voices.uchicago.edu/willett,bGRVPl8AAAAJ
+Rebecca Williams,Univ. of Maryland - Baltimore County,https://www.csee.umbc.edu/people/faculty/rebecca-williams/,NOSCHOLARPAGE

```

Comment:
This is not the correct Rebecca Williams entry in DBLP.

Reply:
corrected to Rebecca M. Williams